es-modules/src/lib/index.js

```diff
@@ -0,0 +1,9 @@
+import noop from './noop';
+import toArray from './to-array';
+import createElement from './create-element';
+
+export {
+  noop,
+  toArray,
+  createElement
+};
```

Comment:
`createElement` is a "named" export. So when you `import` it, you'll need curly braces.

```js
import { createElement } from '../lib';
```